components/Editor.js

```diff
@@ -179,7 +178,7 @@ class Editor extends React.Component {
 
     const prefix = options.filename || this.state.name || 'carbon'
 
-    return this.getCarbonImage({ format, type: 'blob' })
+    return this.getCarbonImage({ format })
```

Comment:
This call is no longer correct. This will now use the `png` path instead of the `blob` path. 

I would update `format = 'blob'` above, and also update where `exportImage` is called in `ExportMenu`